packages/manager/modules/pci/src/projects/project/kubernetes/add/add.controller.js

```diff
@@ -89,11 +91,34 @@ export default class {
         this.goBack(this.$translate.instant('kubernetes_add_success')),
       )
       .catch((error) => {
-        this.CucCloudMessage.error(
-          this.$translate.instant('kubernetes_add_error', {
-            message: get(error, 'data.message'),
-          }),
-        );
+        let errorMessage = get(error, 'data.message');
+        if (
+          get(error, 'data.status') === 412 &&
+          errorMessage.indexOf('[') !== -1 &&
+          errorMessage.indexOf(']') !== -1
+        ) {
+          const errorId = errorMessage.slice(
+            errorMessage.indexOf('[') + 1,
+            errorMessage.indexOf(']'),
+          );
```

Comment:
As we discuss together and in addition of Marie comment it's better to have a dedicated function to manage the errorId.
you can have something like this :

`
getErrorId(error) {
            try {
                let errorMessage = get(error, 'data.message');
                let errorStatus = get(error, 'data.status');
                
                if (errorStatus === 412) {
                    return errorMessage.slice(
                        errorMessage.indexOf('[') + 1,
                        errorMessage.indexOf(']');
                }
                
                return null;
            } catch (e) {
                return null;
            }
          }
`